HelpAssistHer/help-assist-her
# Update go button

## Description
Style the "Go" button 

## Test Plan
1. Go to http://localhost:4000/mini-app
1. The button should look like this:
<img width="464" alt="screen shot 2018-10-29 at 8 07 30 pm" src="https://user-images.githubusercontent.com/6378435/47693286-5a480400-dbb6-11e8-87f5-fff94fd88dc6.png">
3. Hover over the button and it should look like this:

<img width="347" alt="screen shot 2018-10-29 at 8 08 23 pm" src="https://user-images.githubusercontent.com/6378435/47693297-6a5fe380-dbb6-11e8-900c-6c4500244f64.png">
